lighthouse-logger/yarn.lock

```diff
@@ -2,16 +2,17 @@
 # yarn lockfile v1
 
 
-debug@^2.6.8:
+debug@^2.6.9:
```

Comment:
weird that the package.json version and actual version in here got bumped in #9398 but this line didn't 🤷